clang/test/Driver/sycl-device-sanitizer.cpp

```diff
@@ -1,3 +1,7 @@
+// UNSUPPORTED: system-windows
```

Comment:
What is preventing this from being supported on Windows?

Reply:
We don't have plan to support device side sanitizer on windows yet, so it needn't test on windows.

Reply:
There is no indication in the implementation that this is the case.  It would be good to add some restrictions for windows as well as supporting diagnostics to inform the user that `-fsanitize` on Windows is currently not supported.

Reply:
I added a warning diagnostics on windows.

Reply:
Test updated too. I added a new test file for windows.